casper/src/main/scala/coop/rchain/casper/genesis/contracts/RegistryBootstrap.scala

```diff
@@ -0,0 +1,42 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.models.Var.VarInstance.{BoundVar, FreeVar}
+import coop.rchain.models.rholang.implicits._
+import coop.rchain.models._
+import coop.rchain.rholang.interpreter.Runtime._
+
+object RegistryBootstrap {
+  val AST: Par = Par(
+    news = Seq(
+      bootstrap(FixedChannels.REG_LOOKUP2),
+      bootstrap(FixedChannels.REG_INSERT_RANDOM2),
+      bootstrap(FixedChannels.REG_INSERT_SIGNED2)
+    )
+  )
+  private def bootstrap(channel: Par): New =
```

Comment:
Please add a comment describing the equivalent rholang, and newlines around methods/values for readability

Reply:
Done